tests/test_pktgen.py

```diff
@@ -0,0 +1,129 @@
+import logging
+import pytest
+import random
+import re
+from tests.common.helpers.assertions import pytest_assert
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('any'),
+    pytest.mark.device_type('vs')
```

Comment:
HI @svelamal 

Is it necessary run this test on vs?  Or just "pytest.mark.asic('cisco')" is enough ?

+ @yutongzhang-microsoft  for viz